### PR TITLE
fix: Fix project-level app query in app permission.

### PR DESCRIPTION
**Pull request by shuofan**
#### What this PR does / why we need it:
Fix project-level app query in app permission.

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @Effet 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | Fix project-level app query in app permission.             |
| 🇨🇳 中文    |  修复应用权限中的 项目级应用查询            |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/core-services/services/security/process.go [hunk 1]
```diff
@@ -267,9 +267,10 @@ func (a AppPermissionProcess) GetAllRoles(ctx context.Context) ([]string, error)
 	}
 	// get all project-level application
 	_, data, err := a.Adaptor.Db.GetApplicationsByIDs(&dto.OrgID, &dto.ProjectID, nil, &apistructs.ApplicationListRequest{
```
Comment (Effet) [inline modules/core-services/services/security/process.go:269]: already pass &dto.ProjectID, is it nessasery to construct ApplicationListRequest with ProjectID?

**Review by Effet — DISMISSED** [review]

File: modules/core-services/services/security/process.go (hunk 1 above)
Comment (shuofan) [reply modules/core-services/services/security/process.go:269]: we are not using the second params (&dto.ProjectID) , will remove it in future.

Comment (Effet) [reply modules/core-services/services/security/process.go:269]: okay

**Pull request merged**